### PR TITLE
[Code Quality] Optimize the codes in module Nacos Config

### PR DESCRIPTION
**Pull request by brotherlu-xcq**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

[Code Quality] Optimize the codes in module Nacos Config
mainly contains:
- remove Chinese notes
- add the interface notes
- remove the unnecessary var assign

## Brief changelog

no

## Verifying this change

see the changes

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: config/src/main/java/com/alibaba/nacos/config/server/service/datasource/LocalDataSourceServiceImpl.java [hunk 1]
```diff
@@ -50,17 +50,17 @@
  * @author Nacos
  */
 public class LocalDataSourceServiceImpl implements DataSourceService {
-    
+
```
Comment (KomachiSion) [inline config/src/main/java/com/alibaba/nacos/config/server/service/datasource/LocalDataSourceServiceImpl.java:53]: Please do not change the indent.
Please use nacos code style to reformat code.

File: config/src/main/java/com/alibaba/nacos/config/server/service/repository/extrnal/ExternalStoragePersistServiceImpl.java [hunk 2]
```diff
@@ -1307,9 +1307,9 @@ public int aggrConfigInfoCountNotIn(String dataId, String group, String tenant,
                 + " FROM ( SELECT id FROM config_info   ORDER BY id LIMIT ?,?  )"
                 + " g, config_info t WHERE g.id = t.id ";
         PaginationHelper<ConfigInfoWrapper> helper = createPaginationHelper();
-        
+
```
Comment (KomachiSion) [inline config/src/main/java/com/alibaba/nacos/config/server/service/repository/extrnal/ExternalStoragePersistServiceImpl.java:1310]: Same problem with above

File: config/src/main/java/com/alibaba/nacos/config/server/service/repository/extrnal/ExternalStoragePersistServiceImpl.java [hunk 3]
```diff
@@ -1854,8 +1851,6 @@ public ConfigInfoAggr findSingleConfigInfoAggr(String dataId, String group, Stri
         
         try {
             return this.jt.queryForList(sql, new Object[] {dataId, groupId, content}, String.class);
-        } catch (EmptyResultDataAccessException e) {
```
Comment (KomachiSion) [inline config/src/main/java/com/alibaba/nacos/config/server/service/repository/extrnal/ExternalStoragePersistServiceImpl.java:1857]: Why remove it?

File: config/src/main/java/com/alibaba/nacos/config/server/service/repository/extrnal/ExternalStoragePersistServiceImpl.java [hunk 4]
```diff
@@ -2045,8 +2040,6 @@ public void removeTagByIdAtomic(long id) {
         String sql = "SELECT tag_name FROM config_tags_relation WHERE data_id=? AND group_id=? AND tenant_id = ? ";
         try {
             return jt.queryForList(sql, new Object[] {dataId, group, tenant}, String.class);
-        } catch (EmptyResultDataAccessException e) {
```
Comment (KomachiSion) [inline config/src/main/java/com/alibaba/nacos/config/server/service/repository/extrnal/ExternalStoragePersistServiceImpl.java:2048]: Why remove it?

**Review by KomachiSion — CHANGES_REQUESTED** [review]

**Comment by KomachiSion** [comment]
This PR changes so many files, I suggest that you can split this PR to small PR.

For example, you can split with the package. One PR for one package. It will help us review and help you to upgrade.

**Comment by brotherlu-xcq** [comment]
@KomachiSion Thank you for your suggestion, I will close this PR and make the code quality step by step.

**Pull request closed without merging**